lib/grpc/server/adapters/cowboy/handler.ex

```diff
@@ -376,6 +376,12 @@ defmodule GRPC.Server.Adapters.Cowboy.Handler do
     result = server.__call_rpc__(path, stream)
 
     case result do
+      {:ok, _stream, error = %GRPC.RPCError{message: nil, status: status}} ->
+        {:error, %{error | message: GRPC.Status.status_message(status)}}
+
+      {:ok, _stream, error = %GRPC.RPCError{}} ->
+        {:error, error}
+
```

Comment:
I believe more changes are needed so that we get this errors to bubble up without raising. Currently they still raise on like 364.

The idea is that if we manage to bubble them up, then they'll bypass any exception-handling workflow and they'll also follow all interceptors along instead of cutting the workflow dry after the call

Reply:
Surely I'm wrong, but I'm understanding that those exits are because that part of the code is executed with a `spawn_link + Process.flag(:trap_exit, true)`. I have updated the Server module to return an error tuple that in fact is what I think the logger interceptor expects. I'm missing something no?